.github/ISSUE_TEMPLATE/tracking_issue.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Tracking Issue
+about: Track the progress of a specific task or feature
+title: 'Short summary of the task or feature'
```

Comment:
I am wondering why the text needs to be wrapped between \' and \' just for the title field, but not for name and about.

Reply:
I find it's OK to remove them after I test it without `''`. And [syntax-for-issue-forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#markdown-issue-template) provides an example with `''`.  

Since Github won't parse it, I think either would be fine.